aip/general/0193.md

```diff
@@ -47,11 +47,18 @@ error message **must** be considered a backwards-incompatible change.
 **Note:** The [`ErrorInfo`][ErrorInfo] message is the recommended way to send a
 machine-readable identifier.
 
+#### Uniqueness
+
+Each type of detail payload **must** be included only once. For example,
```

Comment:
This sounds like each type of error must have one instance. I.e. one debug, one help, one retryinfo, ... one of each and every one.

Maybe something like:
"If there are multiple types of details, each type **must** appear maximum once"

Reply:
Good catch. Done.